### PR TITLE
Hipp 868 request production access

### PR DESCRIPTION
**Pull request by llmikeyj**

### CODE REVIEW

File: app/views/application/RequestProductionAccessView.scala.html [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+        }
+        <div>
+            <h2 class="govuk-body-l">@messages("requestProductionAccess.instruction")</h2>
+        </div>
+
+        @for(applicationApi <- applicationApis) {
+            <details class="hip-details hip-expander">
+                <summary class="hip-details__summary">
+                    <span class="hip-details__summary-text govuk-heading-s govuk-!-margin-bottom-0">
+                        @{applicationApi.apiDetail.title} usage policy
```
Comment (edgarjimenez) [inline app/views/application/RequestProductionAccessView.scala.html:233]: As per our conversation irl, the styling here is not quite correct

File: app/views/application/RequestProductionAccessView.scala.html [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+            CheckboxesViewModel(
+                form = form,
+                name = "value",
+                legend = Legend(""),
+                items = RequestProductionAccessDeclaration.checkboxItems
+            )
+        )
+
+        @govukButton(
+            ButtonViewModel(messages("requestProductionAccess.button.accept"))
```
Comment (PaulCDurham) [inline app/views/application/RequestProductionAccessView.scala.html:272]: Prototype and Jira ticket specify "Continue" not "Accept"

**Review by PaulCDurham — CHANGES_REQUESTED** [review]
Maybe look back over the view and compare it closely to the prototype. Have you reviewed that with JP?

File: app/views/application/RequestProductionAccessView.scala.html [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+@endpointDetails(endpoint: ApplicationEndpoint, apiDetail: ApiDetail) = {
+    <div class="nhs-body govuk-!-margin-bottom-2">
+        @endpointName(endpoint)
+        @endpointSummary(endpoint, apiDetail)
+    </div>
+}
+
+@descriptionOf(apiDetail: ApiDetail, applicationEndpoint: ApplicationEndpoint) = @{
+    apiDetail.endpoints.filter(endpoint => endpoint.path.equals(applicationEndpoint.path)).flatMap(endpoint => endpoint.methods).filter(endpointMethod => endpointMethod.httpMethod.equals(applicationEndpoint.httpMethod)).map(endpointMethod => endpointMethod.description)
+}
```
[the remark this answers is not in the document]
Comment (PaulCDurham) [reply app/views/application/RequestProductionAccessView.scala.html:168]: Or maybe even summary.getOrElse(description.getOrElse(""))? They're both Option. So favour summary if present. Maybe ask JP

**Review by PaulCDurham — APPROVED** [review]

**Review by edgarjimenez — APPROVED** [review]

**Pull request merged**